manifests/overlays/apps/apps-onprem/starburst-enterprise/query-starburstenterprise-quickstart.yaml

```diff
@@ -43,40 +43,40 @@ spec:
 
-        For example, if the URL for your Starburst web UI is:  
+        For example, if the URL for your Starburst web UI is:
         **https://console-openshift-console.apps.test-cluster.example.com**
 
-        
-        Then the hostname for your Starburst cluster is:  
+
+        Then the hostname for your Starburst cluster is:
         **console-openshift-console.apps.test-cluster.example.com**
 
 
         Confirm the port number for you Starburst cluster.  The default port is **443** for clusters with TLS enabled and is **8080** for clusters with TLS disabled.
       review:
         instructions: |-
           #### To verify you have all the connection details you need:
-          Do you have the hostname copied somewhere for your Starburst Enterprise cluster?  Have you verified the port number for your Starburst Enterprise cluster?  
+          Do you have the hostname copied somewhere for your Starburst Enterprise cluster?  Have you verified the port number for your Starburst Enterprise cluster?
         failedTaskHelp: This task is not verified yet. Try the task again.
       summary:
         success: You have all of the connection details you need to connect to your Starburst Enterprise cluster.  Proceed to the next step.
         failed: Try the steps again.
 
-    - title: Launch JupyterHub from Red Hat OpenShift AI
+    - title: Launch Jupyter from Red Hat OpenShift AI
       description: |-
-        ### To launch a JupyterHub notebook server from within the Red Hat OpenShift AI console:
+        ### To launch a Jupyter notebook server from within the Red Hat OpenShift AI console:
         1. Click **Applications** &#x2192; **Enabled**.
-        2. Find the **JupyterHub** card.
-        3. Click **Launch** on the JupyterHub card to access the JupyterHub **Start a notebook server** page.  A new browser tab will open displaying the **Start a notebook server** page.
+        2. Find the **Jupyter** card.
+        3. Click **Launch** on the Jupyter card to access the Jupyter **Start a notebook server** page.  A new browser tab will open displaying the **Start a notebook server** page.
         4. Select the **Standard Data Science** notebook image and the **Default** container size.
         5. Add the environment variables for **TRINO\_USERNAME**, **TRINO\_PASSWORD**, **TRINO\_HOSTNAME**, and **TRINO\_PORT** using the connection details you obtained in the previous task.
-        6. Click on **Start server** to launch JupyterHub.
+        6. Click on **Start server** to launch Jupyter.
 
         If you have not previously authorized the jupyterhub-hub service account to access your account, the **Authorize Access** page appears prompting you to provide authorization. Inspect the permissions selected by default, and click the **Allow selected permissions** button.
```

Comment:
same here with `jupyterhub-hub`

Reply:
Those are correct right now - starburst docs were submitted by an external contributor and until we get some feedback from them, I will leave them as is.